python/satyaml/VIZARD-ION.yml

```diff
@@ -0,0 +1,17 @@
+name: VIZARD-ION
+alternative_names:
+  - RS68S
+norad: 61749
+data:
+   &tlm AX25 Telemetry:
+     telemetry: ax25
+transmitters:
+   9k6 FSK downlink:
+     frequency: 436.870e+6
+     modulation: FSK
+     baudrate: 9600
+     framing: GEOSCAN
+     deviation: 4800
```

Comment:
Where does the deviation come from? For (G)MSK I would expect 2400 Hz deviation. Has this been measured or published?

Reply:
Also the same question about deviation for RTU-MIREA1 and HORIZON, which got merged on #703 and #698 respectively. I didn't spot this when reviewing those PRs.

I have looked at GEOSCAN-EDELVEIS, which is the other GEOSCAN framing satellite that we already have, and it doesn't have a deviation declared.

Reply:
I have done a measurement.

`gr_satellites TUSUR-GO.yml --rawint16 cut_iq_57600_10788052.wav --samp_rate 57600 --iq --disable_dc_block --dump_path .`

![image](https://github.com/user-attachments/assets/1cf85870-cdd8-4685-8057-5b601c9c2abe)

Reply:
This is a measurement with 2400 deviation 

![image](https://github.com/user-attachments/assets/3d9e8cb5-0c96-4727-a423-4f909ee04e70)

Reply:
This looks good. I agree that the deviation is 4800 Hz.